### ISSUE TITLE
Failed action error 34

### ISSUE THREAD

**Issue by quicoto** [g3rv4/GMFActionDemo#3]
Hello @g3rv4 

I get this error without much information of how to fix it (something wrong with one of the listed instances? something else?)

I just started happening 1 day ago. **Any idea?**

Thank you 

https://github.com/quicoto/HashtagsPull/actions/runs/4132199068/jobs/7140615203

```
Unhandled exception. System.ArgumentNullException: Value cannot be null. (Parameter 'source')
   at System.Linq.ThrowHelper.ThrowArgumentNullException(ExceptionArgument )
   at System.Linq.Enumerable.Where[TSource](IEnumerable`1 , Func`2 )
   at Program.<>c__DisplayClass0_0.<<<Main>$>b__4>d.MoveNext() in /var/src/Program.cs:line 124
--- End of stack trace from previous location ---
   at System.Threading.Tasks.Parallel.<>c__50`1.<<ForEachAsync>b__50_0>d.MoveNext()
--- End of stack trace from previous location ---
   at Program.<Main>$(String[] args) in /var/src/Program.cs:line 83
   at Program.<Main>(String[] args)
/home/runner/work/_temp/ecd7a7a2-f7dd-4868-a0c0-40bd275feb80.sh: line 4:  1739 Aborted                 (core dumped) ./GetMoarFediverse /tmp/tmp.lHzAKiE8Qs/config.json
```

**Comment by g3rv4** [comment]
Thanks for the report :) it's fixed now on 1.0.50 (you don't have to do anything, the next time the action runs it will download the latest file)

**Comment by g3rv4** [comment]
it seems like mastodon.social stopped sharing the posts that are tagged in `/<tag.json>`
